apps/website/postcss.config.js

```diff
@@ -1,15 +1,14 @@
-const { join } = require('path');
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
```

Comment:
⚙️ Consistency: Doesn't newer version require the prefix? Some files of this PR include it and some don't.
```suggestion
import { dirname, join } from 'node:path';
import { fileURLToPath } from 'node:url';
```

Reply:
The prefix isn't required. I do still want to use `node:` everywhere, but I'd prefer to take care of that during ESLint configuration (#14), as this can be autofixed, and there it will also enforce `import path from 'node:path'`.

Reply:
Meaning you want me to fix that instead of you 🤨